src/api/routes/offer.js

```diff
@@ -68,4 +68,12 @@ module.exports = (app) => {
             return next(err);
         }
     });
+
+    router.post("/edit", authMiddleware.isCompanyOrGod, validators.edit, (req, res, next) => {
```

Comment:
Add a middleware validator that checks if the offer is editable in the first place (if not .current() or in grace period <- TODO)

An easy way to check if offer is "current" is to `Offer.find({_id}).current()`. If it returns an offer, it will be current, otherwise it will return null. Additionally, remember that offers that have a publishEndDate in the past (already past the "current" status) cannot be edited as well for history keeping reasons

Reply:
Current indicates if the offer is active?

Reply:
Current indicates that "today" is between publishDate and publishEndDate so it might be visible in search

The offer can however be hidden and stop being shown (so I don't like to use the word "active" for this)